### DISCUSSION TITLE
第三方插件库，无法下载插件，求教解决办法

### DISCUSSION THREAD

**Discussion by MrLi1982** [jxxghp/MoviePilot#2650] [问题咨询]
![QQ_1724222794286](https://github.com/user-attachments/assets/69790ddb-a6fe-43b4-8b86-8766b911de21)
PROXY_HOST： 网络代理；DOH_ENABLE： DNS over HTTPS开关；DOH_DOMAINS： DOH域名清单；GITHUB_TOKEN： Github token都已经按需设置环境变量，同时插件仓库配置也是按照教程！！可是始终就是无法下载插件库的插件，在线求教！！请大神赐教！！

**Comment by MoogleXX** [comment]
同样问题。其他所有设置都已经正常。就是第三方插件库怎么弄都是获取插件库失败。直接第三方插件库地址打开又是正常的。
![_20240823163614](https://github.com/user-attachments/assets/0b41f179-bdae-4b0d-ba78-f76d4369a2a2)

**Comment by Devinaille** [comment]
两位解决了吗，我这也相同情况。
镜像初始化都能正常下载github的数据，但程序自己去获取就不正常

**Reply by MoogleXX** [reply]
没解决。重新搞了个容器。然后把之前数据库复制过来正常能用。。。

**Reply by Devinaille** [reply]
> 没解决。重新搞了个容器。然后把之前数据库复制过来正常能用。。。

测试了是电信网络解析raw.githubusercontent
.com返回0.0.0.0导致的，换个阿里dns就好了
